packaging/linux/run_keybase

```diff
@@ -10,6 +10,30 @@ runtime_dir="${XDG_RUNTIME_DIR:-$HOME/.config}/keybase"
 mkdir -p "$runtime_dir"
 startup_token="$runtime_dir/startup_mode"
 
+# How much shall be started, valid choices are
+#   "service": just the keybase service
+#   "kbfs": the service and the user's kbfs
+#   "gui": service, kbfs, gui
+# Note that the redirector startup is controlled independently.
+startup_level=""
+
+fetch_startup_level() {
+  startup_level_key="startup_level"
+
+  startup_level="gui"
+  if [ -r "$rootConfigFile" ] ; then
```

Comment:
It probably doesn't make sense to refer to `rootConfigFile` here. In the `run_redirector` function where we're using it, we're calling `keybase config -c "$rootConfigFile" ...`, but here we're not passing that option (which is correct, since we'd want to respect the user's own config file).